packages/zosfiles/src/api/methods/download/Download.ts

```diff
@@ -84,8 +84,15 @@ export class Download {
                 reqHeaders = [ZosmfHeaders.X_IBM_BINARY];
             }
 
-            // Get contents of the data set
+            if (options.encoding) {
```

Comment:
Same comment as Get.ts

Reply:
good point - i did this one first and forget when i got to Upload